core/drivers/clk/clk.c

```diff
@@ -235,6 +235,14 @@ struct clk *clk_get_parent(struct clk *clk)
 	return clk->parent;
 }
 
+size_t clk_get_num_parents(struct clk *clk)
+{
+	if (clk_is_clk_elt(clk))
```

Comment:
struct clk_elt and friends are not available yet, they are introduced in the next commit.

Reply:
I've miss something when squashing. I'll fix that at next squash party if you're ok.

Reply:
Fine. :-)